lumps/fdm_deh.lmp

```diff
@@ -73,33 +73,33 @@ par 32 0
 # Text shown on startup in Boom-compatible ports. This is all in
 # STARTUP5, so if a mod replaces STARTUP1-4, it will still be shown.
 STARTUP5============================================================================\nThis is FreeDM, the free content deathmatch FPS.\n\nFreeDM is freely redistributable under the terms of the modified BSD\nlicense. Check out the Freedoom website for more information:\n\n    https://freedoom.github.io/\n============================================================================
-GOTARMOR = picked up a light armor vest.
-GOTMEGA = picked up a heavy armor vest.
-GOTHTHBONUS = got a 1% health bonus.
```

Comment:
Keeping these describe the actual effect (1%) rather than something vague I think is appropriate considering it's a deathmatch WAD. I wanted to do the same for the other health bonuses as well, but it's not possible since they depend on the skill level.